web/client/embeddedTemplate.html

```diff
@@ -105,6 +105,11 @@
             }
         </script>
     </head>
+    <style>
+        .leaflet-top {
+            top: 50px;
+        }
+    </style>
```

Comment:
Is it possible to add this style in the leaflet.less file to target specifically the emebedded pages?

I think we should avoid to add style related to the structure of mapstore inside the html because this needs to be repeated for each html, included the ones of the project derived from mapstore and project templates.

I think we could pass a custom className in the [configuration](https://github.com/geosolutions-it/MapStore2/blob/e5b70a5ca53ef396664b7a761256e483dd03e77a/web/client/product/appConfigEmbedded.js#L17-L21) of the page inside the embedded app:

```js
pages: [{
  name: "mapviewer",
  path: "/:mapId",
  component: require('./pages/MapViewer').default,
  pageConfig: {
    className: 'viewer map-viewer-embedded' // we should keep viewer to ensure previous style are applied
  }
}],
```

- this className should replace the class of the page [here](https://github.com/geosolutions-it/MapStore2/blob/e5b70a5ca53ef396664b7a761256e483dd03e77a/web/client/containers/MapViewer.jsx#L49)

![image](https://user-images.githubusercontent.com/19175505/117140202-6fb4cf00-adad-11eb-8fda-a306ca47d87f.png)

- now if the class show up in the node we can use a selector for the zoom controls in the leaflet.less

```css
.map-viewer-embedded {
 .leaflet-top {
  top: 50px;
 }
}
```